package.json

```diff
@@ -92,6 +92,7 @@
     "patch-package": "6.2.0",
     "performance-now": "2.1.0",
     "postinstall-prepare": "1.0.1",
+    "proportional-scale": "^3.0.0",
```

Comment:
Tiny one of mine: https://github.com/dzucconi/proportional-scale — just a typesafe and better tested version of the resizing function that's already here, basically.

Reply:
oooh interesting